ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-next.yaml

```diff
@@ -10,10 +10,12 @@ build_root:
     tag: stackrox-test-0.3.57
 releases:
   latest:
-    candidate:
+    prerelease:
+      architecture: amd64
       product: ocp
-      stream: nightly
-      version: "4.13"
+      version_bounds:
+        lower: 4.13.0-0
+        upper: 4.14.0-0
```

Comment:
Can you please for my education share docs about this config if they exist?

Reply:
Unfortunately, I wasn't able to find any. I usually find these sorts of fixes by searching through Slack, particularly #forum-testplatform, ripgreping through `ci-operator/configs`, and looking through ci-operator reference found here: https://steps.ci.openshift.org/ci-operator-reference